templates/action-group.json

```diff
@@ -8,14 +8,26 @@
     "alertShortName": {
       "type": "string"
     },
-    "alertEmailReceivers": {
-      "type": "array"
+    "emailReceiverName": {
+      "type": "string",
+      "defaultValue": "emailReceiver",
```

Comment:
does this need to have a defaultValue?
/ what does this do?

Reply:
This is what appears as the name of the action; "Andy" in the attached screenshot
<img width="895" alt="screen shot 2018-07-05 at 15 54 07" src="https://user-images.githubusercontent.com/38751994/42330757-b719cf2a-806b-11e8-8510-ff214b808244.png">

Reply:
right sure, I would suggest removing the default as you can't provide a good default

Reply:
I was originally expecting to have multiple separate emails actions, one for each email recipient, so I was going to use the name of the email recipient as the name of the action.

Now, I guess it can be fixed and overwriteable - "Mailing List" perhaps?

Reply:
Should probably still have a name for the mailing list,
Bulk print alerts